src/legacy/ui/public/agg_types/buckets/significant_terms.js

```diff
@@ -50,13 +51,17 @@ export const significantTermsBucketAgg = new BucketAggType({
     },
     {
       name: 'exclude',
-      type: 'regex',
-      advanced: true
+      type: 'string',
```

Comment:
Could you please add `displayName' property to have localized label like Terms has (https://github.com/maryia-lapata/kibana/blob/master/src/legacy/ui/public/agg_types/buckets/terms.js#L314).

```js
displayName: i18n.translate('common.ui.aggTypes.buckets.significantTerms.excludeLabel', { defaultMessage: 'Exclude' }),
```